app/views/unattended/report_templates/jobs_-_invocation_report_template.erb

```diff
@@ -23,22 +23,26 @@ require:
 - plugin: foreman_remote_execution
 ￼ version: 4.4.0
 -%>
+<%- report_headers 'Host', 'stdout', 'stderr', 'debug', 'Result', 'Finished' %>
 <%- invocation = find_job_invocation_by_id(input('job_id')) -%>
 <%- parts = ["job_invocation.id = #{input('job_id')}"] %>
 <%- parts << input('hosts') unless input('hosts').blank? %>
 <%- search = parts.map { |part| '(' + part + ')' }.join(' AND ') %>
 <%- load_hosts(search: search).each do |batch| -%>
 <%-   batch.each do |host|  -%>
 <%-     task = invocation.sub_task_for_host(host) -%>
+<%-     outputs = { 'stdout' => [], 'stderr' => [], 'debug' => [] }  -%>
 <%-     task.action_continuous_output.each do |output| -%>
-<%-       report_row(
-            host: host.name,
-            result: task.result,
-            type: output['output_type'],
-            message: output['output'],
-            time: format_time(output['timestamp']),
-          ) -%>
+<%-       outputs[output['output_type']] << output['output'] -%>
 <%-     end -%>
+<%-     report_row(
+          'Host': host.name,
+          'stdout': outputs['stdout'],
```

Comment:
Thanks for the comment! I updated the PR suggested way but little differently. I was struggling with concatenation of the future HTML output so after discuss with @ezr-ondrej I take another approach. Instead of having macro that only print correct line break for right output I made the macro that takes array of values and returns string with values that are chained with right line break.